app/controllers/users_controller.rb

```diff
@@ -15,5 +15,30 @@ def show
     @user = User.find(params[:id])
     @ratings = Rating.where(ratee_id: params[:id])
     @rates = Rating.where(rater_id: params[:id])
+
+    @new_messages = unanswered_unrated(@user)
+  end
+
+  def unanswered_unrated(user)
+    unanswered & all_unrated_conversations(user)
+  end
+
+  def all_unrated_conversations(user)
+    b = Conversation.where(
+          Conversation.where(conversations: {recipient_id: user.id})
+          .where(conversations: {sender_id: user.id})
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Space inside { missing.<br>Space inside } missing.